Sources/Aardvark/Bug Reporting/ViewHierarchyAttachmentGenerator.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Copyright 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+@objc(ARKViewHierarchyAttachmentGenerator)
+public final class ViewHierarchyAttachmentGenerator: NSObject {
+
+    // MARK: - Public Static Methods
+
+    /// Captures a textual representation of the current view hierarchy and returns a bug report attachment containing
+    /// plain text data for that representation.
+    @objc
+    public static func captureCurrentHierarchy() -> ARKBugReportAttachment {
+        return generateAttachment(for: UIApplication.shared.windows)
```

Comment:
nit: you don't need this `return` 😉 
```suggestion
        generateAttachment(for: UIApplication.shared.windows)
```

Reply:
Xcode 10 support 😉

Reply:
![giphy](https://user-images.githubusercontent.com/139364/107322369-32180280-6a59-11eb-8db6-c27204c03b8e.gif)

... probably time to drop that? but can be a separate PR.

Reply:
Oh wait, actually I guess I already bumped to Xcode 11 🤔